sdks/python/apache_beam/testing/benchmarks/nexmark/nexmark_launcher.py

```diff
@@ -83,6 +85,8 @@ def __init__(self):
     self.topic_name = self.args.topic_name + self.uuid
     self.subscription_name = self.args.subscription_name + self.uuid
 
+    print('topic_name:', self.topic_name)
```

Comment:
Was this for debugging?

Reply:
Oops. I didn't pop some corrections I had made. Done!